src/VTUFiles.f90

```diff
@@ -465,7 +465,7 @@ SUBROUTINE writepvtu_VTUXMLFileType(fileobj,funit,case,filen,procs,rank)
   INTEGER(SIK) :: i,j,iord
   CHARACTER(LEN=128) :: fname,fmtStr
   !
-  IF((procs>1).AND.rank == 0) THEN
+  IF(rank == 0) THEN
```

Comment:
Didn't we already merge this?  Or am I crazy?

Reply:
We did, it might be that this was the branch I pushed it up on.....but I am surprised it is being shown as a difference

Reply:
Something weird with not having merged master I guess. It seems to recognize that isn't a change now